test/integration/header_integration_test.cc

```diff
@@ -1375,8 +1377,9 @@ TEST_P(HeaderIntegrationTest, TestTeHeaderPassthrough) {
       });
 }
 
-// Validates TE header is stripped if it contains an unsupported value
+// Validates TE header was stripped if it contains an unsupported value
```

Comment:
nit: Your call, but I'd be inclined to use the present tense here as it's describing what the test is *going* to do, not what it has done. But totally your call.